mbedtls/src/ecp/mod.rs

```diff
@@ -421,6 +423,7 @@ mod tests {
     use crate::bignum::Mpi;
     use crate::ecp::{EcGroup, EcPoint};
     use crate::pk::EcGroupId;
+    use super::Result;
```

Comment:
This line is causing your recent Travis CI build to fail: https://app.travis-ci.com/github/fortanix/rust-mbedtls/jobs/570370369